### ISSUE TITLE
Sitemap.xml.gz cache - how to expire

### ISSUE THREAD

**Issue by ingedmundo** [kjvarga/sitemap_generator#361]
I have a project on pre production phase on heroku - I generated the `public/sitemap.xml.gz`  file and downloaded it, then I make a small refactor and re-generated the sitemap on heroku - but the file seems not to be updated.
Long story short: it seems like heroku is serving the file from some sort of cache becaues even if I remote bash delete the file Im still able to pull it from there. Any idea how can I clear cache so I can see the updated sitemap?

**Comment by jrochkind** [comment]
Are you storing the sitemap on S3 or something like that, just trying to store it on the local file system? Storing it on the local file system won't work on heroku; although the particular way you say it doesn't work surprises me, it is known not to work for other reasons -- heroku once a day refreshes your dynos and you will lose anything you wrote to the local file system. 

The README for htis gem says "Supports read-only filesystems like Heroku via uploading to a remote host like Amazon S3". You might want to try storing on an S3 bucket, if you aren't now?

Regardless, this doesn't seem to be about a bug in this gem, but about how heroku works, so you could try asking heroku support.

**Comment by ingedmundo** [comment]
the sitemap lives in `/public` folder, Im not using s3 to serve the file. I understand that Heroku will remove the files I created on the local file system, thats fine for now. What is super weird is that I keep seeing an old file, even when I have re created the file or even when I have removed the file from the public folder.

and yes, you are right this is not related to the gem itself, sorry for that. I was just hopping somebody could point me in the right direction.
